test/base/SpecBase.scala

```diff
@@ -131,6 +133,15 @@ trait SpecBase extends PlaySpec with GuiceOneAppPerSuite with TryValues {
     .set(ClaimAmountAndAnyDeductions, 80).success.value
     .set(FREResponse, FRENoYears).success.value
 
+  val currentYearMinus1UserAnswers = emptyUserAnswers
+    .set(EmployerContributionPage,  EmployerContribution.NoEmployerContribution).success.value
+    .set(TaxYearSelectionPage, Seq(CurrentYearMinus1)).success.value
+    .set(YourAddressPage, true).success.value
+    .set(YourEmployerPage, true).success.value
+    .set(ClaimAmount, 100).success.value
+    .set(ClaimAmountAndAnyDeductions, 80).success.value
+    .set(FREResponse, FRENoYears).success.value
+
```

Comment:
Would it be possible to use the yearsUserAnswers function to create this instead?